src/components/BankTransactions/BankTransactionsBulkActions.scss

```
@@ -0,0 +1,3 @@
+.Layer__category-select--modal {
```

The CSS class name should match the component name:
```suggestion
.Layer__BankTransactionsBulkActions__CategorySelectModal {
```